sycl/include/CL/sycl/detail/pi.h

```diff
@@ -246,8 +247,17 @@ static const uint8_t PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL = 4;
 /// pi_device_binary_struct.DeviceTargetSpec
 ///
 #define PI_DEVICE_BINARY_TARGET_UNKNOWN "<unknown>"
+/// SPIR: standard portable IR for OpenCL 32-bit version
 #define PI_DEVICE_BINARY_TARGET_SPIRV32 "spir"
-#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64";
+/// SPIR: standard portable IR for OpenCL 64-bit version
+/// Compatible with all SPIR 64-bit subarchitectures
+#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64"
+/// x86-64 CPU specific subarchitecture of SPIR 64-bit
+#define PI_DEVICE_BINARY_TARGET_SPIRV64_X86_64 "spir64_x86_64"
+/// GEN GPU specific subarchitecture of SPIR 64-bit
```

Comment:
I'm not an expert in this, but this also looks confusing. AFAIK SPIR/SPIR-V is some generic binaries, not specified for target architecture, what is "GPU specific subarchitecture of SPIR" is unclear to me.

Reply:
The confusion relates to the discussion above, will be fixed simultaneously